### PR TITLE
Cameras list

### PR DESCRIPTION
**Pull request by szlachcic**

### CODE REVIEW

File: css/dropdown.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
+ /* Dropdown Button */
```
Comment (bjsowa) [inline css/dropdown.css]: This file is an artifact that is no longer used and can be removed

File: index.html [hunk 2]
```diff
@@ -5,48 +5,40 @@
...
 		<!-- icons -->
 		<link rel="shortcut icon" href="./img/logo.jpg">
 		<link rel="apple-touch-icon-precomposed" href="./img/logo.jpg">
 		<link rel="manifest" href="./manifest.webmanifest">
-
 		<link rel="stylesheet" href="css/bootstrap.min.css" />
 		<link rel="stylesheet" href="css/style.css" />
 		<link rel="stylesheet" href="css/bootstrap-slider.min.css" />
 		<link rel="stylesheet" href="css/font.css" />
+		<link rel="stylesheet" href="css/dropdown.css" />
```
Comment (bjsowa) [inline index.html]: Remove this

**Review by bjsowa — CHANGES_REQUESTED** [review]

File: js/leo.js [hunk 3]
```diff
@@ -279,10 +298,51 @@ function defaultVideoSrc() {
...
+    publishersClient.callService(request, function(result) {
+	    var publishers = result.publishers;
+
+        if(publishers.length != 0) {
+            var opt = document.createElement('option'); 
+            opt.innerHTML = topicName.replace("/compressed", "");
+            select.appendChild(opt);
+        }
+    });
+}
```
Comment (bjsowa) [inline js/leo.js]: We should ignore topics that don't end with `/compressed`

**Review by bjsowa — CHANGES_REQUESTED** [review]

File: css/dropdown.css (hunk 1 above)
Comment (szlachcic) [reply css/dropdown.css]: done

**Review by bjsowa — APPROVED** [review]

**Pull request merged**